pom.xml

```diff
@@ -214,6 +216,7 @@
         <libpam4j.version>1.11</libpam4j.version>
         <log4j.version>1.2.17</log4j.version>
         <log4j2.version>2.12.0</log4j2.version>
+        <lombok-version>1.16.20</lombok-version>
```

Comment:
Do we need lombok? Looks like it adds some helper annotations that aren't strictly required. It just ends up being another library to have to maintain if we don't need it.

Reply:
[Lombok](https://projectlombok.org/) avoids a lot of boilerplate code - setters, getters, constructors. Many other modules can be so much concise if using Lombok - also avoids human errors.